astropy/coordinates/transformations/affine.py

```diff
@@ -0,0 +1,361 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
```

Comment:
Most of this docstring should go to `__init__.py`.  This submodule is more specialized and that should be clear from its docstring.

Reply:
✅